### ISSUE TITLE
Issue with no /build/index.js

### ISSUE THREAD

**Issue by gregtatum** [stevekinney/octavian#4]
The main entry to your build doesn't seem to be published on npm, so when I require it I don't get anything. I was able to get it generated by manually running the build script and taking out the rm -rf build part of the npm build script.

Directory doesn't exist: https://github.com/stevekinney/octavian/blob/master/package.json#L6

**Comment by SpazzMarticus** [comment]
Same problem here: I had to manually install the build-dependencies and then start the build.

```
cd node_modules/octavian
npm install
```

This tried to run the build task which failed because of windows reasons.
I then started it with:

```
 node_modules/babel/bin/babel.js lib -d build
```

Then the `var Octavian = require('octavian');` works.

**Comment by stevekinney** [comment]
Hmm—I'll take a look at it. I opted for a different build process in 2.1 and I suspect that's the issue.

**Comment by emkay** [comment]
any update on this?

**Comment by stevekinney** [comment]
I just published 2.2.0, which—hopefully—fixes this issue. Sorry for the delay everyone.

**Comment by gregtatum** [comment]
:+1: Sweet, thanks!
